sys/fuchsia/process.txt

```diff
@@ -0,0 +1,22 @@
+# Copyright 2022 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+# See: https://cs.opensource.google/fuchsia/fuchsia/+/main:zircon/vdso/process.fidl
+
+include <zircon/syscalls.h>
+
+resource zx_process[zx_handle]
+
+zx_process_exit(ret_code int64)
+
+zx_process_create(job zx_job, name ptr[in, string], name_len len[name], options const[0], proc_handle ptr[out, zx_process], vmar_handle ptr[out, zx_vmar])
+
+zx_process_start$allargs(proc_handle zx_process, thread_handle zx_thread, entry ptr[in, text[target]], stack intptr, arg1 zx_handle, arg2 intptr)
+
+zx_process_start$arg1(proc_handle zx_process, thread_handle zx_thread, entry ptr[in, text[target]], stack intptr, arg1 zx_handle, arg2 const[0])
+
+zx_process_start$arg2(proc_handle zx_process, thread_handle zx_thread, entry ptr[in, text[target]], stack intptr, arg1 const[0], arg2 intptr)
+
+zx_process_read_memory(proc_handle zx_process, vaddr intptr, buffer ptr[out, array[int8]], size len[buffer], actual ptr[out, int64])
+
+zx_process_write_memory(proc_handle zx_process, vaddr intptr, buffer ptr[in, array[int8]], size len[buffer], actual ptr[out, int64])
```

Comment:
Is it enabled now and not crashing the kernel anymore as previously mentioned in`processes.txt` ?